src/Types.jl

```diff
@@ -245,7 +245,24 @@ Base.:(==)(t1::PackageEntry, t2::PackageEntry) = t1.name == t2.name &&
     t1.uuid == t2.uuid
     # omits `other`
 Base.hash(x::PackageEntry, h::UInt) = foldr(hash, [x.name, x.version, x.path, x.pinned, x.repo, x.tree_hash, x.deps, x.uuid], init=h)  # omits `other`
-const Manifest = Dict{UUID,PackageEntry}
+
+Base.@kwdef mutable struct Manifest
+    julia_version::Union{Nothing,VersionNumber} = Base.VERSION
+    manifest_format::VersionNumber = v"1.0.0" # default to older flat format
```

Comment:
Is the `Manifest` constructor actually ever called without these arguments being explicitly given? If not, I suggest the defaults are removed.

Reply:
It is called, when a manifest file doesn't exist https://github.com/IanButterworth/Pkg.jl/blob/8281b20d232768e0bcc4f28eb85965ffdd8a94e1/src/manifest.jl#L176